tests/utils/test_validation.py

```diff
@@ -26,6 +26,7 @@
     "a/b/c",
     "a.b.c",
     ".a",
+    ":",
```

Comment:
could you add test name like `aa:bb:cc` too ?

Reply:
```suggestion
    ":",
    "aa:bb:cc",
```